tests/integration/test_sign_in.py

```diff
@@ -348,3 +349,21 @@ def test_obfuscate_email(self):
 
         for test in testAddresses:
             self.assertEqual(obfuscate_email(test), testAddresses[test])
+
+    @requests_mock.mock()
+    @patch("frontstage.views.help.is_authorization")
+    def test_sign_in_help_post_who_is_ons(self, mock_request, is_authorization):
+        mock_request.get(url_banner_api, status_code=404)
+        is_authorization.return_value = True
+
+        form = {"option": "ons"}
+        response = self.app.post("/help/info-ons", data=form, follow_redirects=True)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertIn("Who is the Office for National Statistics (ONS)?".encode(), response.data)
+
+        self.assertIn(
```

Comment:
I don't like it, however secure message is on the page in both combinations (authorised or not)